.gitignore

```diff
@@ -20,14 +20,18 @@ save.xml
 
 # Executables
 src/czmq_selftest
-src/makecert
```

Comment:
Strange... did these deliverables disappear, or are they not listed in project.xml?